pkg/output/table.go

```diff
@@ -115,9 +149,18 @@ func (c *tableManager) Output(ctx context.Context, out interface{}) error {
 		if preTableText != "" {
 			pterm.Println(preTableText)
 		}
-		err := table.Render()
-		if err != nil {
-			return err
+		if len(tableData) > 2 {
+			err := table.Render()
+			if err != nil {
+				return err
+			}
+		} else {
+			tableData = transposeTable(tableData)
+			table := pterm.DefaultTable.WithHasHeader().WithData(tableData)
+			err := table.Render()
+			if err != nil {
+				return err
+			}
```

Comment:
This isn't going to work, because what if we call `cone task search` and it only has one result? It would display the other way which a consumer of the output should not have to anticipate for parsing.

You should add a new `OutputOptions` pattern and make an option `Transpose`, then make the if statement based on that. Then for commands that we want to show transposed, we'll add that `OutputOption` to show it horizontally. That way commands that aren't opted in don't show the transposed direction.

Here is an example of the options pattern: https://github.com/ConductorOne/cone/blob/5e805019cdd2fc98dbd1b86c59cf4304346b6473/pkg/uhttp/client.go#L88-L90